javascripts/discourse/api-initializers/full-width.js

```diff
@@ -4,6 +4,8 @@ import { h } from "virtual-dom";
 import { iconNode } from "discourse-common/lib/icon-library";
 
 export default apiInitializer("0.8", (api) => {
+  document.body.classList.add("full-width-enabled");
```

Comment:
```suggestion
  document.body.classList.toggle("full-width-enabled", true);
```

Or does `add` already prevent adding duplicates? (in the test env where initializer is run many times)

Reply:
`add` already does deduplication 👌